maintainers/README.md

```diff
@@ -0,0 +1,114 @@
+# Nixpkgs Maintainers
+
+The *Nixpkgs maintainers* are people who have assigned themselves to
+maintain specific individual packages. We encourage people who care
+about a package to assign themselves as a maintainer. When a pull
+request is made against a package, OfBorg will notify the appropriate
+maintainer(s).
+
+## Reviewing contributions
+
+### Individual maintainer list
+
+When adding users to [`maintainer-list.nix`](./maintainer-list.nix), the following
+checks should be performed:
+
+- If the user has specified a GPG key, verify that the commit is
```

Comment:
Nowadays it is possible to use ssh keys for commit signing.
There's nothing written here about how to add that information to the maintainers list.

Reply:
Good point for a follow-up PR. AFAIK the maintainers-list.nix schema doesn´t support it and would have to be extended as well.

Reply:
Yeah, this section was just moved from https://nixos.org/manual/nixpkgs/stable/#reviewing-contributions-individual-maintainer-list, I don't want to update it just yet, but that's a good future change.

Note that there is also some documentation in the `maintainers-list.nix` file itself.